tests/lib/rules/prop-types.js

```diff
@@ -1486,6 +1486,7 @@ ruleTester.run('prop-types', rule, {
         '  }',
         '}'
       ].join('\n'),
+      settings: {react: {flowVersion: '0.52.0'}},
```

Comment:
Since we've checked this syntax in 0.52, it *must* be a breaking change to default to 0.53.

My strong preference is to initially release a semver-minor with the default as 0.52, and then later release a semver-major that bumps the default to Flow latest.